stageleft/src/lib.rs

```diff
@@ -13,7 +13,13 @@ pub mod internal {
     pub use quote::quote;
     pub use {proc_macro_crate, proc_macro2, syn};
 
-    pub type CaptureVec = Vec<(String, (Option<TokenStream>, Option<TokenStream>))>;
```

Comment:
presumably should still be pub to match the visibility of QuotedOutput. Also a good time to make `(String, (Option<TokenStream>, Option<TokenStream>))` into its own struct perhaps